Modules/ModParser.lua

```diff
@@ -2356,6 +2356,7 @@ local specialModList = {
 		mod("AvoidStun", "BASE", 100, { type = "Condition", var = "UsingFlask" }),
 	},
 	["unaffected by curses"] = { mod("CurseEffectOnSelf", "MORE", -100) },
+	["you are hexproof if you have a (%a+) (%a+) in (%a+) slot"] = function(_, rarity, item, slot) return { mod("AvoidCurse", "BASE", 100, { type = "Condition", var = rarity:gsub("^%l", string.upper).."ItemIn"..item:gsub("^%l", string.upper).." "..(slot == "right" and 2 or slot == "left" and 1) }) } end,
```

Comment:
I was originally trying to make the "you are hexproof" bit a separate thing, but it required more code than I was ready to implement, so this line is needlessly generic.  I might take another crack at it if this doesn't get merged before I find the time

Reply:
Let's take the time to use the right abstraction instead.